app/templates/components/learner-select-field.hbs

```diff
@@ -0,0 +1,6 @@
+{{#if selection.value}}
```

Comment:
I used this select addon that I helped build. It's more flexible in terms of styling, can support placeholders (as an option as well), and fires primary action when one changes the value.